### DISCUSSION TITLE
Trap of too much

### DISCUSSION THREAD

**Discussion by hellcry37** [andrew-codechimp/HA-Battery-Notes#1761] [General]
In my opinion be careful about to not bring into this integration to many things. This is already good for what it should do, if it has to many features it will be harder to maintain, update and finally will be abandoned like many others.

Just my 2 cents!

**Comment by andrew-codechimp** [comment]
I agree totally, I think this integration is feature complete for the majority of users and maintenance will be the main focus unless something is not intrusive to the average user.

I've been developing mobile apps and open source for a very long time now and don't start something unless I feel I can commit to keeping it supported for the long term, I'm only finally killing off some android projects after ten years.
